package.json

```diff
@@ -50,6 +50,7 @@
     "react": "16.1.0",
     "react-dom": "16.1.2",
     "react-redux": "5.0.7",
+    "react-router-dom": "5.1.2",
```

Comment:
What's the reason why we're adding `react-router-dom` as a dependency now?

Reply:
It's a peer dependency of `frontend-auth` now 
https://github.com/edx/frontend-auth/blob/master/package.json#L55

Reply:
Ah, I see. Thanks for the clarification!